src/main/java/Ultron/TaskList.java

```diff
@@ -0,0 +1,114 @@
+package ultron;
+
+import java.util.ArrayList;
+
+import ultron.tasks.Task;
+
+public final class TaskList {
+
+    /** Store the internal list of tasks. */
+    private final ArrayList<Task> list;
```

Comment:
Consider changing the name to better reflect the intent of the variable. Something along the lines of `tasks`.

Reply:
Yes, good suggestion :D